src/main/java/dp/xlsx/DateLabel.java

```diff
@@ -0,0 +1,27 @@
+package dp.xlsx;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Helper for working with date labels in v4 files.
+ */
+public class DateLabel {
+
+    private static final Map<String, String> DATE_FORMAT_REGEXPS = new HashMap<String, String>() {{
+        put("^\\w{3}-\\d{2}$", "MMM-yy");
```

Comment:
I've never seen this syntax before - no idea you could even do this.

Reply:
👍